package.json

```diff
@@ -9,23 +10,31 @@
   "author": "Nathan Faubion <nathan@n-son.com> (https://github.com/natefaubion/)",
   "license": "MIT",
   "repository": "natefaubion/purescript-psa",
+  "files": [
+    "dist/*",
+    "index.js"
+  ],
   "bin": {
     "psa": "index.js"
   },
   "directories": {
-    "test": "test"
+    "test": "testcase"
   },
   "scripts": {
+    "version" : "echo 'export const version = \"v'$npm_package_version'\";' > ./src/Main.js && git add ./src/Main.js",
+    "postversion": "git push && git push --tags",
     "build": "npm run compile",
-    "compile": "psa -c \"src/**/*.purs\" \"bower_components/purescript-*/src/**/*.purs\"",
-    "compile-self": "node index.js -c \"src/**/*.purs\" \"bower_components/purescript-*/src/**/*.purs\"",
-    "pretest": "rm -rf test/output",
-    "test": "node index.js --is-lib=test/bower_components --is-lib=test/lib -o \"test/output\" \"test/*.purs\" \"test/lib/*.purs\" \"test/bower_components/purescript-*/src/**/*.purs\"",
-    "pretest-purs": "rm -rf test/output",
-    "test-purs": "purs -o \"test/output\" \"test/*.purs\" \"test/bower_components/purescript-*/src/**/*.purs\"",
-    "prepublish": "rm -rf output && npm run compile"
+    "compile": "spago bundle --outfile dist/index.js --bundle-type module --platform node",
+    "compile-self": "node index.js -c \"src/**/*.purs\" \".spago/p/**/src/**/*.purs\"",
+    "pretest": "rm -rf testcase/output",
+    "test": "node index.js --is-lib=testcase/bower_components --is-lib=testcase/lib -o \"testcase/output\" \"testcase/*.purs\" \"testcase/lib/*.purs\" \"testcase/bower_components/purescript-*/src/**/*.purs\"",
+    "pretest-purs": "rm -rf testcase/output",
+    "test-purs": "purs compile --output=testcase/output \"testcase/*.purs\" \"testcase/bower_components/purescript-*/src/**/*.purs\"",
+    "prepare": "rm -rf output && npm run compile"
   },
   "devDependencies": {
-    "bower": "^1.7.2"
+    "bower": "^1.8.14",
```

Comment:
I think we need an esbuild dependency for spago bundle to work.

Reply:
Done